config/kubernetes-sigs/sig-autoscaling/teams.yaml

```diff
@@ -0,0 +1,35 @@
+teams:
```

Comment:
Is there a way to create a team for the reviewers or is that solely handled by the `OWNERS_ALIASES` file that's in `kubernetes-sigs/karpenter`?

Reply:
Yea, creating a team would be fine.

Who all you need to be added to the reviewers team? And with what level of access (read is enough?)

Reply:
Read should be enough. Just talking about reviewer-level. This is the list of users from the proposed PR: https://github.com/jonathan-innis/karpenter-core/blob/add-new-member-approvers/OWNERS_ALIASES#L14

Reply:
added `karpenter-reviewers` team